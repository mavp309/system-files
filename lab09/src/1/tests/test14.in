i 50
i 40
i 60
d 45
s 50
s 45
p -1
q 0
