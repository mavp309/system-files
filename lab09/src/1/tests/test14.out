Failed
True
False
60 40 50 
